sound/soc/intel/boards/sof_sdw.c

```diff
@@ -15,9 +15,32 @@
 #include "sof_sdw_common.h"
 
 unsigned long sof_sdw_quirk = SOF_RT711_JD_SRC_JD1;
+static int quirk_override = -1;
+module_param_named(quirk, quirk_override, int, 0444);
```

Comment:
0444? Not writable by anyone? Is this correct?

Reply:
it's what we use for other machine drivers

````
bytcht_es8316.c:module_param_named(quirk, quirk_override, int, 0444);
bytcr_rt5640.c:module_param_named(quirk, quirk_override, int, 0444);
bytcr_rt5651.c:module_param_named(quirk, quirk_override, int, 0444);
````

Reply:
I think it's a feature that we don't want anyone to write to sysfs and change the parameter value, since there's no mechanism to update the value and no notification if the value changes.

Most kernel parameters use the same permissions.